permissions/plugin-fireline.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "fireline"
+paths:
+- "com/qihoo/jenkins/plugins/fireline"
```

Comment:
This does not match the data in the POM. Please see this repository's readme for how this value should be built.

Reply:
<groupId>com.qihoo.jenkins.plugins</groupId>
<artifactId>fireline</artifactId>
<version>1.0</version>
<packaging>hpi</packaging>

Reply:
I have changed the "paths".
Check again. Thank you